.circleci/config.yml

```diff
@@ -34,7 +34,19 @@ jobs:
       - deploy:
           name: Deploy
           command: |
+            # Do not release on Broad Holidays
+            CURRDATE=$(date +%m/%d/%Y)
+             array=("03/12/2021" "4/19/2021" "5/28/2021" "5/31/2021" "7/2/2021" "7/5/2021"
```

Comment:
Some of these dates are formatted with a leading 0 (eg `"03/12/2021"`) and others without (eg  `"4/19/2021"`). Presumably only one of those formats is actually generated by the `date` command above and we should match that in all cases?

Reply:
(possibly relevant for the middle digit too -eg do we get `"7/2/2021"` vs `"07/02/2021"`

Reply:
Yeah, good catch I copy/pasted the wrong group thanks!